backend/contributions/tests/test_is_submittable.py

```
@@ -219,6 +220,93 @@ def test_case_insensitive_filter_value(self):
+        )
+        self.other_retired_type = ContributionType.objects.create(
+            name='Other Retired Type',
+            slug='other-retired-type',
+            description='Also retired',
+            category=category,
+            min_points=1,
+            max_points=10,
+            is_submittable=False,
+        )
+
+    def _make_submission(self, state='pending'):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_make_submission`

(ANN202)